src/stratum/stratumConnection.js

```diff
@@ -330,9 +330,14 @@ export class StratumConnection {
         const { error } = json
         try {
           if (error) {
-            const errorMessage = error.message
+            let errorMessage = error.message
               ? error.message.split('\n')[0]
               : error.code
+            // Check for common words found in spam server transaction broadcast responses
+            const spamCheck = new RegExp(/(security|upgrade|image)/i)
```

Comment:
This check looks a bit too aggressive and may have some false positives. I'd recommend changing to look for "Required security update"

Reply:
"required security update" is how it appears in our most recent reported case, but I've seen screenshots of others with "security update required". I grabbed these keywords and compared them to the list actual electrum error messages here to avoid false positives (link points to the most common errors regarding transactions): https://github.com/bitcoin/bitcoin/blob/cd42553b1178a48a16017eff0b70669c84c3895c/src/rpc/protocol.h#L54-L57